api/v1/constructors.go

```diff
@@ -1074,15 +1076,6 @@ func NewSystemSpec(systemInfo v1info.SystemInfo) (*SystemSpec, error) {
 	}
 
 	spec.VSwitchType = &systemInfo.Capabilities.VSwitchType
-
-	if systemInfo.DRBD != nil {
```

Comment:
Since we are deleting the whole block here, should we make corresponding changes to api/v1/system_types.go by removing struct definition "DRBDConfiguration" and updating struct "SystemStorageInfo"?

Reply:
No we need to just filter the attributes from the deployment-config yaml file